pages/index.js

```diff
@@ -1,21 +1,42 @@
 import Link from "next/link";
+import { useRouter } from "next/router";
+
+const router = useRouter;
 
 export default function HomePage() {
   return (
     <>
       <h1>EasyRev</h1>
-      <Link href="/good-page">
-        <button type="button">Good</button>
-      </Link>
-      <Link href="/middle-page">
-        <button type="button">Middle</button>
-      </Link>
-      <Link href="/bad-page">
-        <button type="button">Bad</button>
+      <Link href="../user-pages/good-page">Good</Link>
+      <br />
+      <Link href="../user-pages/middle-page">Middle</Link>
+      <br />
+      <Link href="../user-pages/bad-page">Bad</Link>
+      <br />
+      <Link href="../login/login">
+        <SVGIcon variant="github" width="50px" />
```

Comment:
This Link is not accessible, please fix!

Reply:
Sorry i dont understand why?

Reply:
It's just a button, there is no accessible text. (see aria labels)